core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/clustering/hazelcast/aws/AWSECSBasedMembershipScheme.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.core.clustering.hazelcast.aws;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.HazelcastInstance;
+import org.apache.axis2.clustering.ClusteringFault;
+import org.apache.axis2.clustering.ClusteringMessage;
+import org.apache.axis2.description.Parameter;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Hazelcast membership scheme AWS Elastic Container Service (ECS).
```

Comment:
```suggestion
 * Hazelcast membership scheme for AWS Elastic Container Service (ECS).
```

Reply:
Fixed in https://github.com/wso2/carbon-kernel/commit/501b3b76015e2474793a89dacbab6418012cc053